SampleMerchant/public/index.html

```diff
@@ -83,9 +78,56 @@ <h1>Sample Merchant</h1>
                   value: '10.00'
                 }
               }
+            ],
+            shippingOptions: [
+              {
+                id: "standard",
```

Comment:
nit: Inconsistent use of single and double quote strings.

Reply:
done